Code/GraphMol/FileParsers/XYZFileWriter.cpp

```diff
@@ -0,0 +1,57 @@
+// $Id$
```

Comment:
No need for this `$Id$` line.
Those are leftover from the days we were using svn.